### ISSUE TITLE
unable to connect jupyter kernel

### ISSUE THREAD

**Issue by joemonvarughese** [spyder-ide/spyder#18622]
### What steps will reproduce the problem?

unable to connect exixsting jupyter kernel to spyder console

### Traceback
```python-traceback
  File "C:\Users\2184743\Anaconda3\envs\spyder\lib\site-packages\spyder\plugins\ipythonconsole\plugin.py", line 959, in create_client_for_kernel
    self._create_client_for_kernel(connection_file, hostname, sshkey,
  File "C:\Users\2184743\Anaconda3\envs\spyder\lib\site-packages\spyder\plugins\ipythonconsole\plugin.py", line 1789, in _create_client_for_kernel
    shellwidget.check_spyder_kernel()
  File "C:\Users\2184743\Anaconda3\envs\spyder\lib\site-packages\spyder\plugins\ipythonconsole\widgets\shell.py", line 256, in check_spyder_kernel
    self.silent_exec_method(code)
  File "C:\Users\2184743\Anaconda3\envs\spyder\lib\site-packages\spyder\plugins\ipythonconsole\widgets\shell.py", line 618, in silent_exec_method
    self._request_info['execute'][msg_id] = self._ExecutionRequest(msg_id,
TypeError: __new__() missing 1 required positional argument: 'hidden'
WARNING:traitlets:No such comm: c3bb523b010711ed8bbe005056b49a35
WARNING:traitlets:No such comm: 57246ca6010811ed9da7005056b49a35
WARNING:traitlets:No such comm: 291a4f3b010911edbf2d005056b49a35
WARNING:traitlets:No such comm: 291a4f3b010911edbf2d005056b49a35
WARNING:traitlets:No such comm: c3bb523b010711ed8bbe005056b49a35
WARNING:traitlets:No such comm: 291a4f3b010911edbf2d005056b49a35
WARNING:traitlets:No such comm: 57246ca6010811ed9da7005056b49a35
WARNING:traitlets:No such comm: 291a4f3b010911edbf2d005056b49a35
WARNING:traitlets:No such comm: c3bb523b010711ed8bbe005056b49a35
WARNING:traitlets:No such comm: 57246ca6010811ed9da7005056b49a35
WARNING:traitlets:No such comm: c3bb523b010711ed8bbe005056b49a35
WARNING:traitlets:No such comm: 57246ca6010811ed9da7005056b49a35
WARNING:traitlets:No such comm: 291a4f3b010911edbf2d005056b49a35
WARNING:traitlets:No such comm: 291a4f3b010911edbf2d005056b49a35
```

## Versions

* Spyder version: 4.2.5 
* Python version: 3.8.13
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  2.0.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  7.22.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  4.4.0 (OK)
keyring >=17.0.0                :  23.4.0 (OK)
nbconvert >=4.0                 :  6.4.4 (OK)
numpydoc >=0.6.0                :  1.2 (OK)
paramiko >=2.4.0                :  2.8.1 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.9.0 (OK)
pygments >=2.0                  :  2.11.2 (OK)
pylint >=1.0                    :  2.12.2 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.2             :  0.3.2 (OK)
qdarkstyle >=2.8;<3.0           :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.3 (OK)
qtconsole >=5.0.3               :  5.3.0 (OK)
qtpy >=1.5.0                    :  2.0.1 (OK)
rtree >=0.8.3                   :  0.9.7 (OK)
setuptools >=39.0.0             :  61.2.0 (OK)
sphinx >=0.6.6                  :  4.4.0 (OK)
spyder_kernels >=1.10.2;<1.11.0 :  1.10.2 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0        :  1.0.2 (OK)
zmq >=17                        :  22.3.0 (OK)

# Optional:
cython >=0.21                   :  None (NOK)
matplotlib >=2.0.0              :  3.5.2 (OK)
numpy >=1.7                     :  1.23.1 (OK)
pandas >=1.1.1                  :  1.4.3 (OK)
scipy >=0.17.0                  :  1.8.1 (OK)
sympy >=0.7.3                   :  None (NOK)

# Spyder plugins:
spyder_notebook                 :  0.3.2 (OK)
```

**Comment by ccordoba12** [comment]
Hey @joemonvarughese, thanks for reporting. This error is already fixed in our latest version (5.3.1).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

If you're trying to connect to another environment, I'm afraid you'll to recreate your environment with conda-forge packages as well because to work with our 5.3.1 version, you need Spyder-kernels 2.3.1, but that version is not available in Anaconda either.

Let us know if that works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
